### PR TITLE
Final change "query" to "select"

### PR DESCRIPTION
**Pull request by ShmuelTreiger**
Some final changing of "query" to "select"

The first commit needs frontend changes

The second commit is gis untested stuff, so @aapeliv should probably look it over

**Backend checklist**
- [x] Formatted my code by running `autoflake -r -i --remove-all-unused-imports src && isort . && black .` in `app/backend`
- [ ] Added tests for any new code or added a regression test if fixing a bug
- [x] All tests pass
- [ ] Run the backend locally and it works
- [ ] Added migrations if there are any database changes, rebased onto `develop` if necessary for linear migration history

**Frontend checklist**
- [ ] Formatted my code with `yarn format && yarn lint --fix`
- [ ] There are no warnings from `yarn lint`
- [ ] There are no console warnings when running the app
- [ ] Added any new components to storybook
- [ ] Added tests where relevant
- [ ] All tests pass
- [ ] Clicked around my changes running locally and it works
- [ ] Checked Desktop, Mobile and Tablet screen sizes

### CODE REVIEW

File: app/proto/search.proto [hunk 1]
```diff
@@ -40,7 +40,7 @@ message SearchReq {
   // * `or` to search for one or another search term
   // * `-` (minus) to ignore a search term
   // in the fuzzy search these have no meaning
-  string query = 1;
+  string statement = 14;
```
Comment (kalvdans) [inline app/proto/search.proto]: I don't think statement is a good name. Also I think we should change the protocol separately from internal changes in the backend.

**Review by kalvdans — CHANGES_REQUESTED** [review]
Revert all .proto changes and make them as a separate pull request.

File: app/proto/search.proto (hunk 1 above)
Comment (ShmuelTreiger) [reply app/proto/search.proto]: "statement" is what SQLA generally uses. SQL_statement?

Comment (kalvdans) [reply app/proto/search.proto]: There's no sql in our protocol between frontend and backend.

Comment (ShmuelTreiger) [reply app/proto/search.proto]: How about `search_string` or `search_term`?

**Review by kalvdans — APPROVED** [review]
Looks good to me! Thanks for switching to proper sql nomeclature :)

**Review by aapeliv — APPROVED** [review]

**Comment by aapeliv** [comment]
LGTM

**Pull request merged**